pkg/registries/registry.go

```diff
@@ -60,6 +66,11 @@ func (c Config) Validate() bool {
 	if c.Name == "" {
 		return false
 	}
+	if c.AuthType == "file" || c.AuthType == "secret" {
```

Comment:
do we need to validate the `AuthType` at all? i.e. if I enter "foobarauth" what will happen?